test/integration/users.test.js

```diff
@@ -0,0 +1,156 @@
+const supertest = require('supertest');
+const { factory } = require('factory-girl');
+
+const dbHelper = require('../helpers/db');
+const usersTestData = require('../data/users');
+const app = require('../../app');
+const { errorMessages } = require('../../app/constants');
+
+require('../setup');
```

Comment:
This line is not necessary, the setup file is automatically called when  for every test and even for every describe when you run the tests.